src/boardlib/api/aurora.py

```diff
@@ -367,31 +365,42 @@ def get_bids_logbook(board, token, user_id):
     sync_results = user_sync(board, token, user_id, tables=["bids"])
     return sync_results["PUT"].get("bids", [])
 
+def bids_logbook_entries(board, user_id, token, db_path=None, aggregate=False):
```

Comment:
Can you explain what the goal is for the aggregate version? I thought the two changes needed for Climbdex were the UUID in the final output and the ability to use a token instead of username/password for auth. It seems like there is quite a bit more than this going on but it's a little hard to follow

Reply:
Sounds good and no worries, happy to help if needed!